plothist/plotters.py

```diff
@@ -63,6 +89,36 @@ def create_variable_registry(variables, path="./variable_registry.yaml", reset=F
 
 
 def get_variable_from_registry(variable, path="./variable_registry.yaml"):
+    """
+    This function retrieves the parameter information for a variable from the variable registry file specified by the 'path' parameter.
+    It loads the variable registry file and returns the dictionary entry corresponding to the specified variable name.
+
+    If the variable registry file does not exist, a `RuntimeError` is raised, unless the default path ("./variable_registry.yaml") is used.
+    In that case, it is assumed that the `create_variable_registry` function has not been executed yet.
```

Comment:
Weird behavior. We should always raise a RuntimeError if the registry does not exist